jhove-installer/src/main/scripts/jhove

```diff
@@ -57,7 +57,7 @@ JHOVE_HOME="$( cd "$(dirname "${SCRIPT}" )" && pwd )"
 cd "${CWD}"
 
 # Create Java class path
-CP="${JHOVE_HOME}/bin/jhove-apps-${JHOVE_VERSION}.jar"
+CP="${JHOVE_HOME}/bin/*"
```

Comment:
These changes may impact people who upgrade JHOVE in-place, without it being obvious that something's gone wrong. Old versions of `jhove-apps-*.jar` will remain in the `bin` directory (as they currently do), but now with multiple JARs on the classpath, the application may just resolve the first `jhove-apps-*.jar` it comes across with the necessary classes in it. If it's ordered alphabetically, it will use the oldest version of the `jhove-apps-*.jar` in the `bin` directory instead of the latest, or previously installed, version.

Reply:
OK, I ran into this issue and the next PR contains a fix for this but you might not like it ;) It [cleans out the bin directory on installation now: https://github.com/openpreserve/jhove/blob/rc/jhove-1.18-rc/jhove-installer/src/main/izpack/install.xml#L64

Reply:
A (possibly) more elegant solution is easy enough by dividing up the classpath a little and using a substitution in the installer instead.

Reply:
Ha, nah, that's not *too* bad. *I* was contemplating purging the whole installation folder except for the `conf` directory. No need to be that rash though. We should still probably provide a warning that `bin` will be purged in an upgrade. Or better yet, can we do a more targeted clean-up? Only files of this pattern: `jhove-apps-*.jar`?

Reply:
Divide it in what way?